src/completion-provider/template-completion-provider.ts

```diff
@@ -173,3 +174,7 @@ function toCompletionItemKind(type: string): CompletionItemKind {
     return CompletionItemKind.Class;
   }
 }
+
+function getTextPrefix({ node }: ASTPath): string {
+  return node && node.original.replace('ELSCompletionDummy', '');
```

Comment:
Well in [line L47](https://github.com/josa42/ember-language-server/blob/d1486e92d1682cedd5cde3628ac762e65a30088e/src/completion-provider/template-completion-provider.ts#L47) `ELSCompletionDummy` add, so it needs to be removed here :man_shrugging: